src/xcode/ENA/ENA/Source/Scenes/TicketValidation/SecondConsent/SecondTicketValidationConsentViewModel.swift

```diff
@@ -0,0 +1,209 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+struct SecondTicketValidationConsentViewModel {
```

Comment:
```suggestion
struct TicketValidationSecondConsentViewModel {
```

Reply:
I think then we need to change the first one as well, it should be called TicketValidationFirstConsentViewModel